src/Modules/Events/EventContainer.js

```diff
@@ -15,22 +17,71 @@ const EventContainer = props => {
     const [serverError, setServerError] = useState(false);
     const [loading, setLoading] = useState(false);
     
+    
+
     useEffect(() => {
         let isSearchData = !!props.location.state;
         if (isSearchData){
-            buildSearchData(props.location.state.searchData);
+            // buildSearchData(props.location.state.searchData);
+            onSubmit(props.location.state.searchData);
             props.history.replace({ state: null });
         }
     });
 
-    const handleSubmit = async query => {
-        if(query) {
+    // const handleSubmit = async query => {
+    //     if(query) {
+    //         setLoading(true);
+    //         let foodBankUri = API_URL.FOODBANK_LIST;
+    //         const { zip_code } = query;
+    //         // Going to use axios for now
+    //         try {
+    //             const resp = await axios.get(foodBankUri, { params: { zip_code } });
+    //             const { data } = resp;
+    //             setFoodBankData(data);
+    //             setFoodBankResponse(true);
+    //             setLoading(false);
+    //         } catch (err) {
+    //             setServerError(true);
+    //             setLoading(false);
+    //         }
+    //     }
+    // };
+
+
+    // const buildSearchData = (data) =>{
+    //     if(Object.keys(data)[0]) {
+    //         let searchDetails = {
+    //             'street': data.searchData.street,
+    //             'zip_code': data.searchData.zip,
+    //         };
+    //         setSearchDetails(searchDetails);
+    //         handleSubmit(searchDetails);
+    //     }
+
+    // };
+
+    const ResourceList = () => {
+        if (foodBankResponse) {
+            return <ResourceListComponent dataToChild = {foodBankData} /> ;
+        }
+        if (serverError) {
+            return <h2>Something went wrong</h2>
+        }
+        return null;
+    };
+
+ const { register, errors,handleSubmit, getValues, watch } = useForm();
+ 
+  const onSubmit = async(payload) => {
+         if(payload) {
             setLoading(true);
             let foodBankUri = API_URL.FOODBANK_LIST;
-            const { zip_code } = query;
-            // Going to use axios for now
+            const { zip_code,lat,long } = payload;
```

Comment:
Does the foodbanks API accept a lat/long parameter? If so, does it do anything with the results? The original issue was related to the address search passing a lat/long parameter into the agencies API and it would respond with a more customized estimated distance field.